packages/cli/src/commands/source/catalog/add.ts

```diff
@@ -0,0 +1,160 @@
+import {UploadBatchCallbackData, CatalogSource} from '@coveo/push-api-client';
```

Comment:
@jpmarceau, I will need your stamp of approval for this new file since this command will print a lot of messages